rkt/rkt.go

```diff
@@ -27,6 +27,7 @@ import (
 	"github.com/coreos/rkt/rkt/config"
 	rktflag "github.com/coreos/rkt/rkt/flag"
 	"github.com/spf13/cobra"
+	_ "github.com/spf13/cobra/doc" // transitive dependency for man page generation
```

Comment:
Spoke with @krnowak about the vendoredApps route; tldr: meant for apps only, but why not. I am fine with either approach.